invenio_app_rdm/records_ui/utils.py

```diff
@@ -56,3 +56,15 @@ def set_default_value(record_dict, value, path, default_prefix="metadata"):
         path = "{}.{}".format(default_prefix, path)
 
     dict_set(record_dict, path, value)
+
+
+def format_date(date):
+    """Format date to display it as MONTH DAY, YEAR.
+
+    :param date: The string containing the date.
+    """
+    try:
+        datetime_obj = datetime.strptime(date, '%Y-%m-%d')
+    except ValueError:
+        return date
+    return datetime_obj.strftime("%B %d, %Y")
```

Comment:
Hmm I really thought we had something to do that already... I guess not :thinking:

Reply:
In Jinja you can use ``dateformat`` filter (see https://invenio-i18n.readthedocs.io/en/latest/api.html#date-time-formatting)

E.g.:

```
{{record.access.embargo.until|dateformat) )}} 
```

In this case, I think we should simply add it to the UI serializer similar to the created/updated dates. See e.g. https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/serializers/ui/schema.py#L109

Reply:
Created a [PR](https://github.com/inveniosoftware/invenio-rdm-records/pull/547) in `invenio-rdm-records` adding the date to the UI serializer, thanks a lot!